content/20.treatments.md

```diff
@@ -517,6 +516,43 @@ Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin
 This is particularly important considering ‘stay in place’ guidance has been implemented in many densely populated cities around the world.
 This measure is likely to limit people’s exposure to sunlight and thus reduce endogenous synthesis of vitamin D, potentially weakening the immune system and increasing the risk of COVID-19 infection.
 
+##### Probiotics
+
+Probiotics are “live microorganisms that, when administered in adequate amounts, confer a health benefit on the host” [@doi:10.1038/nrgastro.2014.66]. 
+As a result of the supposed modest benefits of probiotics against common viral infections and evidence that they can modulate the immune response [@doi:10.4082/kjfm.2013.34.1.2; @doi:10.2174/1381612824666180116163411], it has been hypothesized that probiotics may have therapeutic benefits against SARS-CoV-2 [@doi:10.3389/fpubh.2020.00186].
+
+**Potential Mechanisms.**
+Probiotics and next-generation probiotics, which are more akin to pharmacological grade supplements, have been associated with multiple potential beneficial effects for allergies, digestive tract disorders, and even metabolic diseases through their anti-inflammatory and immunomodulatory effects [@doi:10.1038/nmicrobiol.2017.57; @doi:10.1093/advances/nmy063]. 
+However, the mechanisms by which probiotics affect these various conditions are diverse, heterogeneous, and strain specific [@doi:10.1093/advances/nmy063]. 
```

Comment:
I think it might be better to rephrase the last bit ("diverse, heterogeneous, and strain specific") since it's exactly their words, and just explain how it connects with the text! For example, maybe something like:
```suggestion
However, the mechanisms by which probiotics affect these various conditions would likely differ among strains, with the ultimate effect of the probiotic depending on the heterogenous set of bacteria present [@doi:10.1093/advances/nmy063]. 
```

(This actually may not be what they mean by diverse or heterogeneous? so please take my interpretation with a big grain of salt!)